apps/site/pages/en/about/previous-releases.mdx

```diff
@@ -14,12 +14,28 @@ Production applications should only use _Active LTS_ or _Maintenance LTS_ releas
 
 ![Releases](https://raw.githubusercontent.com/nodejs/Release/main/schedule.svg?sanitize=true)
 
-Full details regarding Node.js release schedule are available [on GitHub](https://github.com/nodejs/release#release-schedule).
+Full details regarding the Node.js release schedule are available [on GitHub](https://github.com/nodejs/release#release-schedule).
 
 ### Commercial Support
 
-Commercial support for versions past Maintenance phase is available through our OpenJS Ecosystem Sustainability Program partner [HeroDevs](https://herodevs.com/).
+Commercial support for versions past the Maintenance phase is available through our OpenJS Ecosystem Sustainability Program partner [HeroDevs](https://herodevs.com/).
 
-## Looking for latest release of a version branch?
+## Looking for the latest release of a version branch?
 
 <DownloadReleasesTable />
+
+## Official versus Community
+
+The Node.js website offers numerous installation methods that allow Node.js to be installed in a non-interactive manner,
+for example, via CLIs, OS package managers (such as `apt`), or Node.js version managers (such as `nvm`).
+
+The Node.js project, in an attempt to popularize and advertise community efforts, has introduced a new Downloads page that lists both Official and Community installation methods, providing more versatility and options for users.
+With this change, we introduced the concept of "Official" and "Community" installation methods. In order to be considered "Official", the installation method must meet the following requirements:
+
+| Requirements                                                                                          |
+| ----------------------------------------------------------------------------------------------------- |
+| New Node.js releases must be available simultaneously upon the official release                       |
+| Project maintainers have a close relationship with Node.js, at least a direct communication way       |
+| Installation method downloads the official binaries bundled by the Node.js project                    |
+| Installation method does **not** build from source or alter the official binaries provided by Node.js |
+| Installation method is the default recommendation for supported environments                          |
```

Comment:
I'm a bit concerned by this as a requirement, but perhaps I'm not understanding it -- this means we'll only have one "official" method per OS?

This feels like it'll be confusing where methods support multiple OSes, e.g. if fnm were "official" on say Windows, but nvm is default on macOS, as an end-user I would think I'd be confused why fnm is not also "official" on macOS and might think it is a different tool with the same name (like nvm for Windows), which it isn't?

Reply:
> this means we'll only have one "official" method per OS?

No. It just means that our code always tries to find the first matching recommended method by the given compatibility constraints. It doesn't mean that YOUR official method will be always the first one showing up, it only means that official methods will be the ones to show up first.

Reply:
> This feels like it'll be confusing where methods support multiple OSes, e.g. if fnm were "official" on say Windows, but nvm is default on macOS, as an end-user I would think I'd be confused why fnm is not also "official" on macOS and might think it is a different tool with the same name (like nvm for Windows), which it isn't?

I guess you're overcomplicating it, but feel free to suggest alternative phrasing

Reply:
@MattIPv4 let me know of my explanation clears out your doubts, but I definitely agree the phrasing here could be better, open for suggestions ;d

Reply:
Ah okay -- I don't think this is a "requirement" then that the method can do in advance to become official, I think it is just a "perk" of being in the official list, that you'll get preferential ordering/listing on the downloads page?

Reply:
That is correct! On the last table maybe it made more sense, any suggestions for rephrasing? This should probably be removed from the table and added as a paragraph below the table

Reply:
Ah cool! I would probably just remove the line and not worry, I think it is implied from the downloads page already where the official heading is before the community heading in the dropdown.